app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantDetailViewModel.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Google LLC
+ * Copyright 2019 Google LLC
```

Comment:
typically copyright isnt updated

Reply:
@nic0lette @tiembo how about?

Reply:
Yeah, typically we only update the date for new files. (So I guess that's "we only set the date on new files"). Leaving it as 2018 would probably be better.

Reply:
dealed